plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -265,6 +265,13 @@ func init() {
 		},
 	})
 	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "ttl-controller"},
+		Rules: []rbac.PolicyRule{
+			rbac.NewRule("update", "patch").Groups(legacyGroup).Resources("nodes").RuleOrDie(),
```

Comment:
since this logically needs list/watch on nodes, include it in the role (even though it's using a shared informer)